src/AppBundle/Repository/PageviewsRepository.php

```diff
@@ -0,0 +1,97 @@
+<?php
+declare(strict_types=1);
+
+namespace AppBundle\Repository;
+
+use AppBundle\Model\EventWiki;
+use DateTime;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+
+/**
+ * A PageviewsRepository is used to fetch data from the pageviews API.
+ * @link https://wikimedia.org/api/rest_v1/#/Pageviews_data
+ */
+class PageviewsRepository
+{
+
+    public const GRANULARITY_HOURLY = 'hourly';
+
+    public const GRANULARITY_DAILY = 'daily';
+
+    public const GRANULARITY_MONTHLY = 'monthly';
+
+    /** @var string Base URL for the REST endpoint. */
+    protected $endpointUrl = 'https://wikimedia.org/api/rest_v1/metrics/pageviews';
+
+    /**
+     * Given a Mediawiki article and a date range, returns a daily timeseries of its pageview counts.
+     * @link https://wikimedia.org/api/rest_v1/#!/Pageviews_data/get_metrics_pageviews_per_article_project_access_agent_article_granularity_start_end
+     * @param EventWiki $eventWiki
+     * @param string $article Page title with underscores. Will be URL-encoded.
+     * @param string $granularity The time unit for the response data, either GRANULARITY_DAILY or GRANULARITY_MONTHLY.
+     * @param DateTime $startTime
+     * @param DateTime $endTime
+     * @return string[][]
+     */
+    public function perArticle(
```

Comment:
We generally use verb function names, so shouldn't this be something like getPerArticle()?

Reply:
Done.